src/homepageExperience/components/steps/arduino/InitializeClient.tsx

```diff
@@ -68,9 +70,16 @@ export const InitializeClient: FC<OwnProps> = ({
 
   useEffect(() => {
     dispatch(getBuckets())
-    dispatch(getAllResources())
   }, []) // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
```

Comment:
What's the rationale behind breaking this into a separate `useEffect` hook? Shouldn't we run both only when `dispatch` changes (which is never - so they will just run once on page load)? That would let us remove the `react-hooks/exhaustive-deps` warning too.

Reply:
I broke these up purely for consistency with the other similar pages, which also have them broken up. Additionally, see https://github.com/influxdata/ui/pull/5489 for the linter warnings.

Reply:
That sounds fine to me, as long as it's the same as the other initialize CLI pages.